src/request/components/RequestDetailPanelWebServices.tsx

```diff
@@ -0,0 +1,130 @@
+import { IRequestDetailWebServicesRequestState } from '../stores/IRequestDetailWebServicesRequestState';
+import { MasterDetailTriPanel } from './MasterDetailTriPanel';
+
+import _ = require('lodash');
+import React = require('react');
+import classNames = require('classnames');
+
+interface IServiceMessagesProps {
+    requests: IRequestDetailWebServicesRequestState[];
+    selectedRequestId: string;
+    onSelectRequest: (requestId: string) => void;
+}
+
+/**
+ * React class to display console messages
+ */
+class ServiceMessages extends React.Component<IServiceMessagesProps, {}> {
+    public render() {       
+        // get child items
+        var requestItems = [];
+        for (var i = 0; i < this.props.requests.length; i++) {
+            const request = this.props.requests[i];
+            
+            requestItems.push(
+                <tr key={request.id} onClick={e => this.props.onSelectRequest(request.id)} className={classNames({
+                        'selected': request.id === this.props.selectedRequestId
+                    })}> 
+                    <td>{request.url}</td>
+                    <td>{request.statusCode}</td>
+                    <td>{request.method}</td>
+                    <td>-</td>
+                    <td>-</td>
+                    <td>-</td>
+                    <td>-</td>
+                </tr>);
+            
+        }
+        
+        return (
+            <table className='table table-bordered table-striped table-selectable tab-web-services-table'>
+                <thead>
+                    <tr className='table-col-title-group'>
+                        <th><span className='table-col-title'>Name/Path</span></th>
+                        <th width='10%'><span className='table-col-title'>Status</span></th>
+                        <th width='10%'><span className='table-col-title'>Method</span></th>
+                        <th width='10%'><span className='table-col-title'>Protocol</span></th>
+                        <th width='10%'><span className='table-col-title'>Type</span></th>
+                        <th width='10%'><span className='table-col-title'>Duration</span></th>
+                        <th width='20%'><span className='table-col-title'>Timeline</span></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {requestItems}
+                </tbody>
+            </table>
+        );
+    }
+}
+
+/**
+ * React class to display service details
```

Comment:
I didn't do any renaming of the components; at the moment those panels show only the request/response headers but more "details" will eventually shown (e.g. body, cookies, etc.).